src/js/actions.js

```diff
@@ -31,7 +31,13 @@ export const Actions = {
     ON_SYSTEM_CAPABILITY_UPDATED: "ON_SYSTEM_CAPABILITY_UPDATED",
     ON_APP_SERVICE_DATA: "ON_APP_SERVICE_DATA",
     ON_PUT_FILE: "ON_PUT_FILE",
-    SET_GLOBAL_PROPERTIES: "SET_GLOBAL_PROPERTIES"
+    SET_GLOBAL_PROPERTIES: "SET_GLOBAL_PROPERTIES",
+    UPDATE_AVAILABLE_APPSTORE_APPS: "UPDATE_AVAILABLE_APPSTORE_APPS",
+    UPDATE_INSTALLED_APPSTORE_APPS: "UPDATE_INSTALLED_APPSTORE_APPS",
+    APPSTORE_ADD_APP_PENDING_SET_APP_PROPERTIES: "APPSTORE_ADD_APP_PENDING_SET_APP_PROPERTIES",
```

Comment:
Remove appstore prefixes here as well

Reply:
removed in https://github.com/smartdevicelink/generic_hmi/pull/240/commits/b79ab73fcfd760a30437c25d53d1c6665ee29526